.gitlab-ci.yaml

```diff
@@ -0,0 +1,19 @@
+stages:
+  - build
+
+build-push-master:
```

Comment:
It looks like we will only used tagged version of this image, so what is this step for?

Reply:
While only tagged version of image should be used for gpii-infra, `latest` tag is also built on commits to master -> this is mostly for convenience and to keep `latest` tag in sync to latest code in `master`.